### ISSUE TITLE
Éric Vyncke's No Objection on draft-ietf-ccamp-otn-topo-yang-18

### ISSUE THREAD

**Issue by italobusi** [haomianzheng/IETF-ACTN-YANG-Model#175]
> Thanks for the work done in this document. The topic is very specific and
> contains mainly a long YANG module, i.e., my review has focused on the non-YANG
> parts as I am trusting the OPS AD and YANG doctors for the YANG part.
> 
> Talking about YANG, I am afraid that this I-D exhibits an issue of doing
> 'delta' model by augmenting an existing one: the tree view is too long and is
> mostly useless. Nothing that can be addressed in this I-D of course.
> 
> Comments:
> 
> In section 1, is "server-layer network" actually the usual "underlay"? If so,
> why not using the more common IETF terminology. Else, it is worth defining it.
> 

- [x] @danielkinguk : could you please help us here?

Authors: See our reply on May 14

https://mailarchive.ietf.org/arch/msg/ccamp/tau2ULswlq9Tf2hr06BXCXZ9C8w/

> Also, s/captures topology related information/captures topology-related
> information/ ?
> 

Authors: Fixed

- [x] @haomianzheng : I think this is a straightforward change to XML

> `Optical Transport Networks (OTN)` the OTN acronym is expanded twice in this
> section ;-)
> 

Authors: Fixed

- [x] @haomianzheng : I think this is a straightforward change to XML

See: https://mailarchive.ietf.org/arch/msg/ccamp/RzzVpzoAVGtzlCfs6McA7zrXAQs/

---

- [x] @italobusi : reply to the comments after draft-ietf-ccamp-otn-topo-yang-19 is uploaded

**Comment by italobusi** [comment]
> Thanks for the work done in this document. The topic is very specific and
> contains mainly a long YANG module, i.e., my review has focused on the non-YANG
> parts as I am trusting the OPS AD and YANG doctors for the YANG part.
> 
> Talking about YANG, I am afraid that this I-D exhibits an issue of doing
> 'delta' model by augmenting an existing one: the tree view is too long and is
> mostly useless. Nothing that can be addressed in this I-D of course.
> 
> Comments:
> 
> In section 1, is "server-layer network" actually the usual "underlay"? If so,
> why not using the more common IETF terminology. Else, it is worth defining it.
> 

- [ ] @danielkinguk : could you please help us here?

> Also, s/captures topology related information/captures topology-related
> information/ ?
> 

- [ ] @haomianzheng : I think this is a straightforward change to XML

> `Optical Transport Networks (OTN)` the OTN acronym is expanded twice in this
> section ;-)
> 

- [ ] @haomianzheng : I think this is a straightforward change to XML

**Comment by danielkinguk** [comment]
Thanks. It is a recurring issues. I'd suggest responding to Eric with:

>>
With CCAMP documents, we typically use the terms "server-layer" and "client-layer" to be consistent with optical transport, specifically the ITU-T SG15 terminology. I agree that we use "underlay" and "overlay" in the IETF, but that's typically used for packet domain discussions. 

I just checked the recent publications and found several other I-Ds that all use the term "server-layer". Therefore, we would prefer to continue to use the terms as described. You make a valid point, so maybe a definition early in the document would be useful. 

"A transport network is a server-layer network designed to provide connectivity services for a client-layer connectivity. This facilitates client traffic to be carried transparently across the server-layer network resources."
<< 

Actually, I think we just make the small edit anyway and say that's what we did.
